### ISSUE TITLE
Add upload support

### ISSUE THREAD

**Issue by eburnette** [uniconnector/chatgpt-ui#3]
The code currently uses VITE_API_UPLOAD to upload data but this was not included in the repo.

If it's possible to support uploads without a backend, that would be even better.

**Comment by uniconnector** [comment]
Yes, the backend service for uploading and parsing PDF files has not been uploaded yet. We did write a demo service before, but it is not yet complete enough, so we have opened up the UI first. Upload a copy after I improve it.

**Comment by eburnette** [comment]
What do you have in mind for the upload service? There are client-side libraries that can parse PDF files; are you planning to do more like keeping the files for later reference or splitting them up into chunks, embedding them, and putting the embeddings in a vector database?

**Comment by uniconnector** [comment]
The initial idea of the upload service was to allow users to handle uploading and parsing themselves, and return the parsed content to the front-end for processing.
As this project is still positioned as a UI project, it is not yet planned to handle all backend functions and save them to the database.
